test/e2e/framework/util/job/make.go

```diff
@@ -0,0 +1,80 @@
+package job
```

Comment:
I think `resource.go` would be a better name for this file, what do you think?

Reply:
or `fixtures.go`?

Reply:
I think fixtures is a better one. :)